ERCS/erc-6538.md

```diff
@@ -27,65 +27,61 @@ This contract defines an `ERC5564Registry` that stores the stealth meta-address
 The contract is specified below. A one byte integer is used to identify the stealth address scheme. This integer is used to differentiate between different stealth address schemes. A mapping from the scheme ID to it's specification is maintained at [this](../assets/eip-5564/scheme_ids.md) location.
 
 ```solidity
-pragma solidity ^0.8.17;
+pragma solidity ^0.8.20;
```

Comment:
Since we're specifying an exact contract, we should specify an exact compiler version too. Feel free to update this suggestion with whichever version is actually being used
```suggestion
pragma solidity 0.8.23;
```

Reply:
Update in https://github.com/garyghayrat/ERCs/pull/2/commits/0163a5a14a3b9517802953c5b6c7a5b68903cd4a